### ISSUE TITLE
Build fail on IOS : AmplitudeIntegration.h:2:9: error: include of non-modular header inside framework module 

### ISSUE THREAD

**Issue by theochampion** [claimsforce-gmbh/flutter-segment#70]
Hi there,

After installing segment like specified in the segment.com docs:

```
// in ios/Podfile

pod 'Analytics', '~> 3.0'
```

I get a build error on IOS:

```
Launching lib/main.dart on iPhone 8 in debug mode...
lib/main.dart:1
Xcode build done.                                            4.5s
Failed to build iOS app
Error output from Xcode build:
↳
    objc[5016]: Class AMSupportURLConnectionDelegate is implemented in both /usr/lib/libauthinstall.dylib (0x1faced160) and /System/Library/PrivateFrameworks/MobileDevice.framework/Versions/A/MobileDevice (0x1191582b8). One of the two will be used. Which one is undefined.
    objc[5016]: Class AMSupportURLSession is implemented in both /usr/lib/libauthinstall.dylib (0x1faced1b0) and /System/Library/PrivateFrameworks/MobileDevice.framework/Versions/A/MobileDevice (0x119158308). One of the two will be used. Which one is undefined.
    * BUILD FAILED *
Xcode's output:
↳
    Command CompileSwift failed with a nonzero exit code
    While building module 'Segment_Amplitude' imported from /Users/mr/Documents/dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_segment-3.1.2/ios/Classes/FlutterSegmentPlugin.m:5:
    In file included from <module-includes>:1:
    In file included from /Users/mr/Documents/dev/zigg/ios/Pods/Target Support Files/Segment-Amplitude/Segment-Amplitude-umbrella.h:13:
    /Users/mr/Documents/dev/zigg/ios/Pods/Segment-Amplitude/Pod/Classes/SEGAmplitudeIntegration.h:2:9: error: include of non-modular header inside framework module 'Segment_Amplitude.SEGAmplitudeIntegration': '/Users/mr/Documents/dev/zigg/ios/Pods/Headers/Public/Analytics/SEGIntegration.h' [-Werror,-Wnon-modular-include-in-framework-module]
    #import <Analytics/SEGIntegration.h>
            ^
    While building module 'Segment_Amplitude' imported from /Users/mr/Documents/dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_segment-3.1.2/ios/Classes/FlutterSegmentPlugin.m:5:
    In file included from <module-includes>:1:
    In file included from /Users/mr/Documents/dev/zigg/ios/Pods/Target Support Files/Segment-Amplitude/Segment-Amplitude-umbrella.h:14:
    /Users/mr/Documents/dev/zigg/ios/Pods/Segment-Amplitude/Pod/Classes/SEGAmplitudeIntegrationFactory.h:2:9: error: include of non-modular header inside framework module 'Segment_Amplitude.SEGAmplitudeIntegrationFactory': '/Users/mr/Documents/dev/zigg/ios/Pods/Headers/Public/Analytics/SEGIntegrationFactory.h' [-Werror,-Wnon-modular-include-in-framework-module]
    #import <Analytics/SEGIntegrationFactory.h>
            ^
    2 errors generated.
    /Users/mr/Documents/dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_segment-3.1.2/ios/Classes/FlutterSegmentPlugin.m:5:9: fatal error: could not build module 'Segment_Amplitude'
    #import <Segment_Amplitude/SEGAmplitudeIntegrationFactory.h>
     ~~~~~~~^
    3 errors generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Analyzing workspace
    note: Constructing build description
    note: Build preparation complete
Could not build the application for the simulator.
Error launching application on iPhone 8.
Exited (sigterm)
```

The error seems to come from FlutterSegmentAmplitude. Which is even weirder given I disabled Amplitude integration in the settings:

```plist
	<key>com.claimsforce.segment.ENABLE_AMPLITUDE_INTEGRATION</key>
	<false />
```
Any ideas?

**Comment by theochampion** [comment]
@esarbanis By looking at [your PR](https://github.com/claimsforce-gmbh/flutter-segment/pull/49) on installation via dart code I take it you are familiar with the way dependencies are built on IOS.

Any ideas of what could be going on here? ;)

**Comment by MaiKaY** [comment]
Hi @theochampion,
we just released a new version `3.1.3`. Could you please try and confirm that the issue is solved after upgrading to version `3.1.3`?
